zellij-server/src/tab.rs

```diff
@@ -2400,6 +2400,29 @@ impl Tab {
             }
         }
     }
+
```

Comment:
Hey! Thanks for submitting this PR! I like the functionality that this enables, but I think we could clean things up a little by having something like `Visible(bool)`, with `Visible(true)` replacing `Active` and `Visible(false)` replacing `InActive`.

That way we could clean up these duplicated functions and keep the number of new Events minimal! :)

Hopefully you won't need to use this hack for your timer in the near future, but this will be useful even after that I think :)

Reply:
This is a nice idea for merging two events into one. 
Currently, I use this mechanism to save and restore the state across multiple instances of a plugin.

Reply:
Nice! That's what I thought :)